src/workerd/api/util.c++

```diff
@@ -290,14 +288,4 @@ void maybeWarnIfNotText(jsg::Lock& js, kj::StringPtr str) {
           "\". The result will probably be corrupted. Consider "
           "checking the Content-Type header before interpreting entities as text."));
 }
-
-kj::String fastEncodeBase64Url(kj::ArrayPtr<const byte> bytes) {
-  auto expected_length = simdutf::base64_length_from_binary(bytes.size(), simdutf::base64_url);
-  auto output = kj::heapArray<char>(expected_length);
-  auto actual_length = simdutf::binary_to_base64(
-      bytes.asChars().begin(), bytes.size(), output.asChars().begin(), simdutf::base64_url);
-  KJ_ASSERT(expected_length == actual_length);
-  return kj::String(kj::mv(output));
```

Comment:
Seems likely this might be the issue?  I think `kj::String` has an internal assert to require that any string pointer content be null-terminated.  Might be able to fix it by allocating a slightly bigger array and adding the terminator before passing to `kj::String`.

Reply:
You're absolutely right. Yes, I'll follow up in a new pull-request to fix this. Let's fix edgeworker first!

Reply:
Thanks!